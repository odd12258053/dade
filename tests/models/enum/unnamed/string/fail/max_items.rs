use dade::model;
#[model]
enum TestModel {
    Value(
       #[field(max_items = 2)]
        String
    ),
}
fn main() {}
