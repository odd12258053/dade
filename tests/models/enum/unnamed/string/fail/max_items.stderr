error: Support condition is min_length, max_length, alias, default and validate
 --> tests/models/enum/unnamed/string/fail/max_items.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^
  |
  = note: this error originates in the attribute macro `model` (in Nightly builds, run with -Z macro-backtrace for more info)
